mirascope/cli/utils.py

```diff
@@ -151,7 +174,15 @@ def get_prompt_versions(version_file_path: str) -> VersionTextFile:
 
 
 def check_prompt_changed(file1_path: Optional[str], file2_path: Optional[str]) -> bool:
-    """Checks if the given prompts have changed."""
+    """Compare two prompts to check if the given prompts have changed.
+
+    Args:
+        file1_path (str): The path to the first prompt.
```

Comment:
drop the `(str)` here. The type comes from the function definition and shouldn't also be here.

Reply:
Please make sure that any other instances of this are also cleaned up elsewhere.